### PR TITLE
*: promote the boltdb-freelistType from experimental to official

### PR DESCRIPTION
**Pull request by WIZARD-CXY**
fixes issue #https://github.com/etcd-io/etcd/issues/11258

I think a changelog is necessary to tell user how to operate maybe in CHANGELOG-3.5 and CHANGELOG-3.4? @jingyih @gyuho  WDYT

### CODE REVIEW

**Comment by jingyih** [comment]
> I think a changelog is necessary to tell user how to operate maybe in CHANGELOG-3.5 and CHANGELOG-3.4?

This change will be part of 3.5. So we should add to CHANGELOG-3.5.

**Comment by jingyih** [comment]
Please also update `etcdmain/help.go`

The current process of adding / updating a config flag is quite manual. It is easy to miss updating some file.

**Comment by WIZARD-CXY** [comment]
@jingyih fixed. Indeed the etcd's configuration flag is many now.

**Comment by WIZARD-CXY** [comment]
I will add it to CHANGELOG-3.5

**Comment by jingyih** [comment]
--- FAIL: TestLessorGrant (0.01s)
    lessor_test.go:87: lease ID expected 1, got 2

Not related but might be a new flaky test.

File: etcdmain/help.go [hunk 1]
```diff
@@ -200,8 +200,8 @@ Experimental feature:
     Duration of time between cluster corruption check passes.
   --experimental-enable-v2v3 ''
     Serve v2 requests through the v3 backend under a given prefix.
-  --experimental-backend-bbolt-freelist-type 'array'
-    ExperimentalBackendFreelistType specifies the type of freelist that boltdb backend uses(array and map are supported types).
+  --backend-bbolt-freelist-type 'map'
```
Comment (jingyih) [inline etcdmain/help.go]: move to line 72?

Comment (WIZARD-CXY) [reply etcdmain/help.go]: ok

**Pull request merged**